test/com/google/enterprise/adaptor/opentext/OpentextAdaptorTest.java

```diff
@@ -3717,6 +3802,12 @@ public void configure(Config config) {
     @Override
     public void setServer(OpentextAdaptor.CwsServer server) {
     }
+
+    @Override
+    public String getAuthenticationToken(
+        DocumentManagement documentManagement) {
```

Comment:
I wanted this to fix on one line so badly I checked it both times this method is implemented. ;-)